Traceback (most recent call last):
  File "/home/matts/phd/implementation/ode_decomp/oslo_control_2.py", line 1168, in <module>
    optimize_start(rebalancing_cost, bounce_cost, run_price=False)##, default_epoch=default_epoch, default_prices=default_prices)
  File "/home/matts/phd/implementation/ode_decomp/oslo_control_2.py", line 1102, in optimize_start
    res, last_vector_iter, new_prices, new_cell_levels, profit, profits, regret, reb_cost = run_control_period_sa(start_hour,end_hour,
  File "/home/matts/phd/implementation/ode_decomp/oslo_control_2.py", line 556, in run_control_period_sa
    ares, lastres, trajectories, last_vector_iter, total_reward, profits, regret, arrivals, bounces = run_control(model_data, traj_cells, SOLVER, 0.5, current_vector=first_vec_iter, time_length=float(end_hour-start_hour))
  File "/home/matts/phd/implementation/ode_decomp/oslo_control_2.py", line 240, in run_control
    twrap = [[multiprocessing.Array(ctypes.c_double, n_time_points+1) for j in range(model_data.n_cells*2)] for i in range(model_data.n_cells)]
  File "/home/matts/phd/implementation/ode_decomp/oslo_control_2.py", line 240, in <listcomp>
    twrap = [[multiprocessing.Array(ctypes.c_double, n_time_points+1) for j in range(model_data.n_cells*2)] for i in range(model_data.n_cells)]
  File "/home/matts/phd/implementation/ode_decomp/oslo_control_2.py", line 240, in <listcomp>
    twrap = [[multiprocessing.Array(ctypes.c_double, n_time_points+1) for j in range(model_data.n_cells*2)] for i in range(model_data.n_cells)]
  File "/usr/lib/python3.10/multiprocessing/context.py", line 141, in Array
    return Array(typecode_or_type, size_or_initializer, lock=lock,
  File "/usr/lib/python3.10/multiprocessing/sharedctypes.py", line 93, in Array
    lock = ctx.RLock()
  File "/usr/lib/python3.10/multiprocessing/context.py", line 73, in RLock
    return RLock(ctx=self.get_context())
  File "/usr/lib/python3.10/multiprocessing/synchronize.py", line 187, in __init__
    SemLock.__init__(self, RECURSIVE_MUTEX, 1, 1, ctx=ctx)
  File "/usr/lib/python3.10/multiprocessing/synchronize.py", line 57, in __init__
    sl = self._semlock = _multiprocessing.SemLock(
OSError: [Errno 28] No space left on device
